inst/htmljs/animint.js

```diff
@@ -1781,12 +1781,17 @@ var animint = function (to_select, json_file) {
       return selected_values;
   };
   
-    
+  var counter=-1;    
```

Comment:
Update selector url is a function that is used to update the url at the bottom of selection menu